lib/npm-install.js

```diff
@@ -1,14 +1,13 @@
-const logger = require('@blackbaud/skyux-logger');
 const spawn = require('cross-spawn');
-const fs = require('fs-extra');
+const logger = require('@blackbaud/skyux-logger');
 
 /**
  * Runs npm install for a specific package
  * @name npmInstall
  */
 function npmInstall(settings) {
-  const message = logger.promise('Running npm install (can take several minutes)');
 
+  let messageText = 'Running npm install. This step can take several minutes.';
```

Comment:
In other console logs, we're wrapping the command in back ticks. Curious if we need to make it consistent?

Reply:
I can.  Within this file; however, none of strings are wrapped in backticks.  I've typically followed the pattern of only using backticks if it needs to be a string literal which has an expression to be evaluated.